content/vendor/packaging/ingress.md

```diff
@@ -87,7 +87,7 @@ kind: Ingress
 metadata:
   name: example-application-ingress-embedded
   annotations:
-    kots.io/exclude: '{{repl or (ConfigOptionEquals "enable_ingress" "1" | not) (not IsKurl) }}'
+    kots.io/exclude: '{{repl or (ConfigOptionEquals "enable_ingress" "1" | not) IsKurl }}'
```

Comment:
Was this a mistake? It used to *exclude* the embedded version if `not IsKurl`. Now it will select the non-embedded version if `IsKurl`. I thought what was here before was correct (though the double negative makes it confusing).

Reply:
thanks! this is incorrect. reverting.